feat(lba-2445): refacto recruiter intentions

alanlr opened the pull request:
https://tableaudebord-apprentissage.atlassian.net/jira/software/projects/LBA/boards/14?assignee=unassigned%2C712020%3A9a003bb4-d766-4edc-b4ab-ecd6529e3353&selectedIssue=LBA-2445

alanlr committed:
feat: engistrement application_id et message_id pour les intentions

alanlr committed:
feat: nouveaux champs

alanlr committed:
feat: migration

alanlr committed:
feat: en bonne voie

alanlr committed:
fix: bad field name

alanlr committed:
fix: bad field

alanlr committed:
feat: màj schéma

alanlr committed:
feat: tests ok

alanlr committed:
Merge branch 'main' into feat/lba-2445-refacto-recruiter-intention

alanlr committed:
fix: test v1

alanlr committed:
fix: typage

alanlr committed:
feat: grrr ... tests sans email

alanlr committed:
fix: test snapshot

alanlr commented:
:rocket:

kevbarns commented on server/src/migrations/20250206175132-refacto-intentions-recruteurs.ts:
```diff
@@ -0,0 +1,45 @@
+  })
+
+  // nettoyage collection recruitment_intentions_mail
+  await getDbCollection("recruiter_intention_mails").deleteMany({})
```
- retirer le modèle et utilise directement le connecteur db dans la migration
- manque un await ligne 27
- utiliser asyncForEach

alanlr committed:
feat: suppression collection recruiter_intention_mails

alanlr committed:
fix: cleaning

alanlr committed:
feat: cleaner

alanlr committed:
Merge branch 'main' into feat/lba-2445-refacto-recruiter-intention

alanlr committed:
Merge branch 'main' into feat/lba-2445-refacto-recruiter-intention

alanlr commented:
:rocket:

alanlr committed:
Merge branch 'main' into feat/lba-2445-refacto-recruiter-intention

kevbarns reviewed (approved):
👍

guilletmarion committed:
Update mail-candidature.mjml.ejs

guilletmarion committed:
Update mail-candidature-partenaire.mjml.ejs

MAJ de wording car de nombreux recruteurs répondent négativement en cliquant sur ce bouton

guilletmarion committed:
Update mail-candidature-spontanee.mjml.ejs

MAJ de wording car de nombreux recruteurs répondent négativement en cliquant sur ce bouton
